README.md

```diff
@@ -34,6 +36,41 @@ This will create the icon fonts in the `build` directory. Additionally CSS and H
 
 _**NOTE**: Prior builds will be removed when a new build is run. Make sure to copy out any files needed before running a new build._
 
+#### In a Vagrant Virtual Machine ####
+
+You can also build the fonts in a Vagrant VM that has all the required libraries.  Before you can do this, you will need
+to:
+
+1. Install Vagrant and VirtualBox.
+2. Install the [Vagrant CI Plugin](https://github.com/gpii-ops/vagrant-gpii-ci) using a command like: `vagrant plugin install vagrant-gpii-ci`
+
+Once you have satisifed these requirements, you can build the fonts in a VM using commands like:
+
+```bash
+vagrant up
+vagrant ci test
```

Comment:
Do the commands have to be called `vagrant ci test`. It's not something that I'd expect to actually generate the font. I'd expect it to do testing as opposed to execution.

Reply:
We're using @amatas' excellent [Vagrant CI Plugin](https://github.com/amatas/vagrant-gpii-ci/tree/NOJIRA-review) for this. You can give jobs any name you want, and from reading the [implementation of our .vagrant.yml](https://github.com/the-t-in-rtf/infusion-icons/blob/ef429fadce3391b461036c38e3e4944483dabf64/.vagrant.yml#L15) above, this is probably just a cut and paste typo in the documentation, since the job is actually called <code>build</code>. There doesn't seem to be a <code>test</code> job at all.

Reply:
Fair point, I will try it with `build`.

Reply:
So, I see the confusion here.  There was previously both a `test` _stage_ and _job_ that.  You can rename either, and I did both so that they more accurately reflect the purpose of the stage and job.  The `test` subcommand provided by the plugin is not configurable from within the configuration file, and is hard-coded as `test` here and various other places in the Vagrant CI plugin:

https://github.com/amatas/vagrant-gpii-ci/blob/NOJIRA-review/lib/vagrant-gpii-ci/command/ci.rb#L21